### PR TITLE
fix(config.js): disable tracing on config set

### PR DESCRIPTION
**Pull request by ranrib**

### CODE REVIEW

File: src/config.js [hunk 1]
```diff
@@ -109,6 +109,10 @@ module.exports.setConfig = function setConfig(configData) {
         config.token = configData.token;
     }
 
+    if (configData.isEpsagonDisabled) {
```
Comment (galbash) [inline src/config.js:112]: why not just `disable`?

**Review by galbash — CHANGES_REQUESTED** [review]
naming question

File: src/config.js (hunk 1 above)
Comment (ranrib) [reply src/config.js:112]: `isEpsagonDisabled` is already defined but not working

**Review by galbash — APPROVED** [review]

**Pull request merged**